src/test/java/games/strategy/engine/config/LobbyPropertyReaderTest.java

```diff
@@ -0,0 +1,59 @@
+package games.strategy.engine.config;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class LobbyPropertyReaderTest {
+  private LobbyPropertyReader testObj;
+
+  /**
+   * Sets up an example lobby property file with some fake values, then sets up a property reader test object
+   * pointed at this file.
+   */
+  @Before
+  public void setup() throws IOException {
+    final File testFile = File.createTempFile("testing", "tmp");
+    testFile.deleteOnExit();
```

Comment:
For future reference, the JUnit `TemporaryFolder` rule is a useful alternative to this construct.